lms/static/js/staff_debug_actions.js

```diff
@@ -127,4 +136,8 @@ $(document).ready(function() {
         StaffDebug.rescore($(this).parent().data('location-name'), $(this).parent().data('location'));
         return false;
     });
+    $courseContent.on("click", '.staff-debug', function() {
+        StaffDebug.check_sudo_and_view_modal($(this).data('element-id'), $(this).data('is-sudo'));
+        return false;
+    });
```

Comment:
Is this trying to enforce sudo mode for the staff-debug information? If so:

1. it doesn't look secure, since it's relying on a client-side check that a) sudo mode is active, and b) what should happen based on the sudo mode. Both of these can be spoofed by the end-user
2. sudo mode should never be active when looking at courseware